### DISCUSSION TITLE
Copilot Not suggesting any inline code and normally too

### DISCUSSION THREAD

**Discussion by ninetynin** [community/community#12584] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/91336664/157384131-4c50b578-2669-4e31-acf6-b46a2ff6b997.png)
First of all in old days github copilot logo used to turn into loading symbol whenever i type something and it shows the suggestions later
but these days copilot is not suggesting any inline code so i used ctrl+enter to see all possible results and it is struck in 0/10 forver 
i tried to see online but there is no issue till now

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by ninetynin** [reply]
oops my bad forget to keep it
here...
`[ERROR] [promptlib proxy] [2022-03-08T06:01:50.783Z] {}
[ERROR] [promptlib proxy] [2022-03-08T06:01:51.378Z] {}
`

**Reply by davecheney** [reply]
Can you post more output please

**Reply by ninetynin** [reply]
> Can you post more output please
```

[ERROR] [promptlib proxy] [2022-03-08T06:01:50.783Z] {}
[ERROR] [promptlib proxy] [2022-03-08T06:01:51.378Z] {}
```

no change I tried spending more time and typing more programs but there is no change.... I tried enabling and disabling and still no change for it too
Should I use live share or meet to show it to you?
Or maybe try reinstalling vsc completely? because i already tried for copilot extension reinstalling

**Comment by ninetynin** [comment, marked as answer]
nvm idk what happened but its working now
mazic
anyway thanks 
still i have one doubt tho @davecheney 
is issues and discussions different or are they same...??
and like there is no way to close the discussion once its over right?
idk why i didnt google it  nvm
thanks anyway

**Reply by davecheney** [reply]
@anand0249 i'm glad its working for you now. Nothing was changed, so 🤷 

> is issues and discussions different or are they same...??

they are different

> and like there is no way to close the discussion once its over right?

there doesn't appear to be a way to close a discussion, an admin can lock them.

**Reply by ninetynin** [reply]
yeah thanks 👍

**Comment by amitabe** [comment]
I'm also having the exact same issue - Copilot is stuck on `Synthesizing 0/10 solutions` and I see only these errors in the output of github Copilot:
```
[ERROR] [promptlib proxy] [2023-01-10T21:23:51.876Z] {}
[ERROR] [promptlib proxy] [2023-01-10T21:23:51.877Z] {}
```

would appreciate help on this, as it's getting me crazy

**Reply by ninetynin** [reply]
Try reinstalling the extension or like maybe try nightly if ur using stable and vice versa
